_posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Pulp 3.11 is now Generally Available!
+author: Melanie Corr
+tags:
+  - release
+---
+
+For a full list of bug fixes and changes that occurred as part of this release, see the [changelog](https://docs.pulpproject.org/pulpcore/changes.html) for this release in the Pulpcore documentation.
+
+## New in Pulp 3.11
+
+Let's take a look at the main features of this release.
+
+### Proxy Authorization Updates
+In previous versions of Pulp, the only way to specify a proxy username and password was to append them in the `proxy_url` fields.
+
+With this release, there are two new fields: `proxy_username` and `proxy_password` for remotes that you can use to enter your credentials. As a result of this change, you can no longer specify credentials as part of the `proxy_url` field.
+  A data migration will move the proxy auth information on existing remotes to the new fields. [#8167](https://pulp.plan.io/issues/8167)
+
+### Worker timeout is no longer hardcoded
+
+In previous versions of Pulp, worker timeout was hardcoded as 30 seconds. Depending on the system, and the I/O of Postgresql, this hardcoded value caused problems.
+
+As part of this release, a new setting `WORKER_TTL` has been added. You can specify the interval, in seconds, that you consider a worker as missing after its last heartbeat. The default value is 30 seconds.
+
+You can edit the `WORKER_TTL` setting in `settings.py`. [#8291](https://pulp.plan.io/issues/8291)
+
+### Systems with existing Artifacts will undergo an automatic update as part of this release
```

Comment:
I would probably move this section's text into the "Removed `md5` and `sha1` from ALLOWED_CONTENT_CHECKSUMS" section or at least have this follow that section.